css/properties/background-attachment.json

```diff
@@ -90,6 +90,7 @@
               ],
```

Comment:
It's out of range for the diff suggestion, but we could also make a correction for Safari desktop. I suspect the right thing is to have a single object (like Chrome etc.) with `{ "version_added": "3.1" }`. The local notes and misbehavior do not seem to apply to desktop Safari.

Reply:
I'll open another PR for that after verifying.